### ISSUE TITLE
Canceling a Incasso Contract

### ISSUE THREAD

**Issue by jurvangeel** [saproto/saproto#350]
Hee Beheer,

I have a small issue, Members (and for that reason the board as well) can't cancel their incasso contract. 
Shouldn't that legally be possible?

situation; I recieved a mail form an ethernal member who wants to stay a member but doesn't want an incasso contract

**Comment by jonathanjuursema** [comment]
We forced all members to have a withdrawal contract after discussing with you at the beginning of the association year. ;)

If you would like another situation, please let us know. If so, please state if (and if so, when) having a contract is required, and if there have to be things that users/members cannot do while not having a withdrawal contract.

**Comment by jonathanjuursema** [comment]
Also, in the meantime, you can just set a bogus withdrawal contract with invalid details.

**Comment by jonathanjuursema** [comment]
Added label feedback as we're waiting on input from @jurvangeel.

**Comment by jurvangeel** [comment]
Ill use the bogus information, this is a too specific request

**Comment by jonathanjuursema** [comment]
Sure! If you want to have us change anything about the withdrawal authorizations, just comment here again or open a new issue. :)
